addons/mail/static/src/widgets/form_renderer/form_renderer_tests.js

```diff
@@ -922,6 +922,40 @@ QUnit.test('Attachments that have been unlinked from server should be visually u
     );
 });
 
+QUnit.test('The chatter should be clear when creating a new record after having displayed a chatter for an existing record', async function (assert) {
```

Comment:
"chatter being cleared" is too technical for a name. Test name should just say what we want functionally (the part after "when creating a new..." is good, it's just the start to change)

also we usually keep test title lowercase (first char)